### DISCUSSION TITLE
Why can't the vehicle return to the original point?

### DISCUSSION THREAD

**Discussion by HuJunnj** [openTCS/opentcs#120] [General]
Why can't the vehicle return to the orFor example, ABCD forms A closed loop, so you can go from point A to point BCD but you can't go from point BCD to point A. Anybody know why?iginal point?

**Comment by swltr** [comment]
@HuJunnj You may want to provide a concrete plant model for that question.

**Comment by freekevinCode** [comment]
Questions without context are meaningless.
